gateway/crates/gateway-binary/src/main.rs

```diff
@@ -33,15 +23,15 @@ fn main() -> anyhow::Result<()> {
         .expect("installing default crypto provider");
 
     let args = self::args::parse();
-    let mut config = args.config()?;
+    let config = args.config()?;
 
     let runtime = runtime::Builder::new_multi_thread()
         .enable_all()
         .thread_name(THREAD_NAME)
         .build()?;
 
     runtime.block_on(async move {
-        let otel_tracing = if std::env::var("__GRAFBASE_RUST_LOG").is_ok() {
+        let telemetry = if std::env::var("__GRAFBASE_RUST_LOG").is_ok() {
```

Comment:
do we still need this?

Reply:
Current logs are unreadable IMHO for debugging... :/